internal/uuidutils/uuidutils.go

```diff
@@ -0,0 +1,19 @@
+package uuidutils
```

Comment:
Do we really need this when we have `strfmt.IsUUID()`? https://pkg.go.dev/github.com/go-openapi/strfmt#IsUUID. This may be an opportunity for refactoring.